### PR TITLE
Adds big eyes as a marking.

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds a new marking that makes your eyes bigger.

## Why It's Good For The Game

Special snowflake eyes.

## Changelog
:cl:
add: Nanotrasen no longer refuses to hire people with larger than average eyes.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm [hunk 1]
```diff
@@ -75,6 +75,13 @@
 	default_color = "F00"
 	affected_bodyparts = HEAD
 	always_color_customizable = TRUE
+	
+/datum/body_marking/other/drake_eyes
```
Comment (Azarak) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/body_markings/body_markings.dm]:
```suggestion
/datum/body_marking/other/big_eyes
```

**Comment by Ghommie** [comment]
Take a look at the icondiffbot diffs, you are also deleting other stuff.

**Comment by ShadeAware** [comment]
Something something anime Catgirls funny

**Comment by Azarak** [comment]
Oh what Ghommie said

**Comment by ghost** [comment]
whoops I accidentally modified an old version of the other_markings.dmi file

**Comment by ghost** [comment]
#1877

**Pull request closed without merging**